.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+## Description
+This PR…
+
+
+## Screenshots
```

Comment:
If it's a UI feature that isn't entirely static, then I usually do a quick screen recording and save as an animated GIF. Maybe invite this by renaming heading to "Screenshots / Screen recording"

Add text: "If your PR contains interactive or otherwise animated UI components, feel free to add a short screen recording in the form of an animated GIF"